cmd/xk6/main.go

```diff
@@ -156,13 +98,12 @@ func runBuild(ctx context.Context, args []string) error {
 		cmd := exec.Command(output, "version")
 		cmd.Stdout = os.Stdout
 		cmd.Stderr = os.Stderr
-		err = cmd.Run()
-		if err != nil {
-			log.Fatalf("[FATAL] %v", err)
+		if err := cmd.Run(); err != nil {
+			return fmt.Errorf("executing k6 %v", err)
```

Comment:
Originally, there was log.Fatal here, which resulted in an exit status of 1. Currently, an exit status of 0 is generated in case of an error, which is a problem from an automation perspective.

Reply:
This will be caught by the error handling when invoking `runDev` and generate a `log.Fatal`